employees/tests/test_views.py

```diff
@@ -181,16 +196,22 @@ def test_custom_report_list_view_should_redirect_to_another_month_if_month_switc
     def test_custom_report_list_view_should_not_display_reports_from_different_month_than_selected(self):
         current_date = datetime.datetime.now().date()
         other_report = ReportFactory(
-            date=current_date + relativedelta(months=+1), author=self.user, project=self.report.project
+            date=current_date + relativedelta(months=+1), author=self.user, project=self.second_report.project
         )
         yet_another_report = ReportFactory(
-            date=current_date + relativedelta(years=-1), author=self.user, project=self.report.project
+            date=current_date + relativedelta(years=-1), author=self.user, project=self.second_report.project
         )
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
         self.assertNotContains(response, other_report.description)
         self.assertNotContains(response, yet_another_report.description)
 
+    def test_form_task_activity_is_initialized_with_latest_activity(self):
+        response = self.client.get(self.url)
```

Comment:
Wouldn't it make tests simpler to add creating of extra report just here, instead of adding it in `setup` and modifying other tests?

Reply:
Ok, good point. One thing needs to be changed in `setUp` though. I will add named `task_activities` for `self.report` just to make sure it is different from one in newly created report.